external/llvh/lib/Support/Unix/Program.inc

```diff
@@ -266,6 +270,7 @@ static bool Execute(ProcessInfo &PI, StringRef Program,
   }
 #endif
 
+#if !(defined(TARGET_OS_TV) && TARGET_OS_TV)
```

Comment:
I'm not sure this is the right approach. ifdef-ing out part of this function is risky, we now have a control path through the function that doesn't return anything.

If we do need to compile this, I would not be opposed to ifdef-ing out the whole body on tvOS, and using `llvh::report_fatal_error` to trigger a runtime crash.